diff -urN ettercap-0.7.3.0/configure ettercap-0.7.3/configure
--- ettercap-0.7.3.0/configure	2007-08-27 20:24:18.000000000 +0530
+++ ettercap-0.7.3/configure	2007-08-27 23:29:42.000000000 +0530
@@ -31109,7 +31109,7 @@
 #define DEBUG 1
 _ACEOF
 
-      DEBUG_FLAGS="-g -ggdb -Wmissing-prototypes -Wno-uninitialized -Werror -Wformat-security -Wsign-compare -Wall"
+      DEBUG_FLAGS="-g -ggdb -Wmissing-prototypes -Wno-uninitialized -Wformat-security -Wsign-compare -Wall"
       echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
       ac_ec_debug=yes
diff -urN ettercap-0.7.3.0/configure.in ettercap-0.7.3/configure.in
--- ettercap-0.7.3.0/configure.in	2007-08-27 20:24:18.000000000 +0530
+++ ettercap-0.7.3/configure.in	2007-08-27 23:29:29.000000000 +0530
@@ -328,7 +328,7 @@
 [ case "$enableval" in
    yes)
       AC_DEFINE(DEBUG, 1)
-      DEBUG_FLAGS="-g -ggdb -Wmissing-prototypes -Wno-uninitialized -Werror -Wformat-security -Wsign-compare -Wall"
+      DEBUG_FLAGS="-g -ggdb -Wmissing-prototypes -Wno-uninitialized -Wformat-security -Wsign-compare -Wall"
       AC_MSG_RESULT(yes)
       ac_ec_debug=yes
       ;;
