# Description: specify file permission in open call

Index: ettercap-0.7.3-1.4ubuntu1/src/ec_log.c
===================================================================
--- ettercap-0.7.3-1.4ubuntu1.orig/src/ec_log.c	2009-07-10 16:11:16.427060161 -0400
+++ ettercap-0.7.3-1.4ubuntu1/src/ec_log.c	2009-07-10 16:11:42.789785610 -0400
@@ -190,7 +190,7 @@
       if (fd->cfd == NULL)
          SEMIFATAL_ERROR("%s", gzerror(fd->cfd, &zerr));
    } else {
-      fd->fd = open(filename, O_CREAT | O_TRUNC | O_RDWR | O_BINARY);
+      fd->fd = open(filename, O_CREAT | O_TRUNC | O_RDWR | O_BINARY, 0664);
       if (fd->fd == -1)
          SEMIFATAL_ERROR("Can't create %s: %s", filename, strerror(errno));
    }
